### PR TITLE
GameDB: Minor Changes for Band Hero, Formula One 2001 and Jonny Moseley Mad Trix

### PR DESCRIPTION
**Pull request by MrCK1**
List of changes:

- Updated compatibility ratings for **Formula One 2001** from 3 to 5 and removed FMV patch in NTSC-U. Ratings were also added to missing regions. 

- Added compatibility ratings for **Band Hero**  - game runs at 5 currently :smiley:

- Updated compatibility ratings for **Jonny Moseley Mad Trix** from 2 to 5

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -4617,14 +4618,8 @@ Region = NTSC-U
...
 Region = NTSC-U
-Compat = 3
+Compat = 5
 XgKickHack = 1
-[patches = 272F1C14]
-	comment=patches by Plot and Ref Skip videos
-	patch=0,EE,0010aafc,word,00000000
-	patch=0,EE,0010ae2c,word,00000000
-	patch=0,EE,00230010,word,00000000
-[/patches]
```
Comment (avih) [inline bin/GameIndex.dbf]: Any info on why that it's not needed anymore?

**Comment by avih** [comment]
Are these compatibility changes already on our online compatibility list? if yes, then typically once in a while we update the whole db with all the updates from the online compatibility list.

If it's not updated at the online compatibility list, then you should first update the info at the forum with details (which pcsx2 version, etc, when did it get fixed/improved), and then wait for the online compatibility list to get updated, and then update `GameIndex.dbf`.

**Comment by ssakash** [comment]
> If it's not updated at the online compatibility list, then you should first update the info at the forum with details (which pcsx2 version, etc, when did it get fixed/improved), and then wait for the online compatibility list to get updated, and then update GameIndex.dbf.

Does PCSX2 repository automatically fetch the data from online compatibility list (or) is it a manual process ? If it's the latter then I don't see the disadvantage on merging this commit.

File: bin/GameIndex.dbf (hunk 1 above)
Comment (refractionpcsx2) [inline bin/GameIndex.dbf]: More than likely, I couldn't say without checking though.

**Comment by refractionpcsx2** [comment]
> Does PCSX2 repository automatically fetch the data from online compatibility list (or) is it a manual process ? If it's the latter then I don't see the disadvantage on merging this commit.

Updating it is a manual affair. We can automatically export the DB from the forum but we don't bother, we only did it for the initial one.  Generally updates are done directly to github like this.

File: bin/GameIndex.dbf (hunk 1 above)
Comment (prafullpcsx2) [inline bin/GameIndex.dbf]: skip video patches aren't needed anymore and game was fixed in https://github.com/PCSX2/pcsx2/commit/ad61503d8311f113ab1fc07d3955f9f576c4f11e

Comment (refractionpcsx2) [inline bin/GameIndex.dbf]: Are you implying I fixed something? You must be mistaken @prafullpcsx2  :P

But cool, if that fixed it, then it is no longer required.

**Comment by avih** [comment]
Ouch @refractionpcsx2  sorry, I accidentally edited your comment instead of replying to it :/

It should really warn me if I'm about to edit someone else's message :/

**Comment by refractionpcsx2** [comment]
That's fine, it makes me look intelligent :D

**Comment by avih** [comment]
Anyway, for the record, I'm adding re-posting what I accidentally edited (without editing it again :) ):

ref:
Updating it is a manual affair. We can automatically export the DB from the forum but we don't bother, we only did it for the initial one. Generally updates are done directly to github like this.


avih:
Semi automatic. AFAIK the online compatibility list should be updates and in sync with the forum. It can be exported as CSV file.

I also wrote a tool which takes this CSV file, compares its data to GameIndex.dbf and updates the latter.

I use this tool occasionally, such as here 6a63a92 .

But anyway, I didn't object to the changes, I just asked if it's in sync and comes from our online compatibility list, or from the forum, etc, and if yes, say so at the commit message, and if no then it should.


avih:
blah blah


ref:
I'm smart!

**Comment by refractionpcsx2** [comment]
> avih:
> blah blah
> 
> ref:
> I'm smart!

xD

File: bin/GameIndex.dbf (hunk 1 above)
Comment (FlatOutPS2) [inline bin/GameIndex.dbf]: Was this patch even necessary after the 0.9.X versions? The only issue on PAL version video's was just that they were black screens with hardware renderers until the blackFMV PR.

Comment (MrCK1) [inline bin/GameIndex.dbf]: @FlatoutPS2 you said in the forum that it shouldn't be necessary after the most recent commit here (can't embed on mobile)
http://forums.pcsx2.net/Thread-Formula-One-2001-SCES-50004-E--30588?pid=377960&highlight=Formula+One+2001#pid377960

**Comment by MrCK1** [comment]
@FlatoutPS2 [you said in the forum](http://forums.pcsx2.net/Thread-Formula-One-2001-SCES-50004-E--30588?pid=377960&highlight=Formula+One+2001#pid377960) that it shouldn't be necessary anymore. I tested without the patches and had no problems.

Is this PR good to go then?

**Comment by avih** [comment]
@MrCK1 if it's it in sync with the compatibility forum and the online compatibility list at the site, then just add this info to the commit message and note why the patch is not needed anymore or/and since when.

If it's not in sync with the online compatibility list (for all your changes, not just the one with your patch), then the next time I update `GameIndex.dbf` from the online list, it will revert these values to whatever the online list has.

The compatibility list is not easy to manage, but we still try keep the various places it's used at in sync. So the order is:
- Update the compatibility forum with the new info if it's not there yet.
- Wait till the online compatibility list shows the new compatibility status.
- Update `GameIndex.dbf` manually or automatically such that it's in sync with the public list.

**Comment by MrCK1** [comment]
@avih Jonny Moseley and Band Hero are up to date in the online PCL. I'll fix Formula One 2001 later tonight

**Comment by avih** [comment]
> @avih Jonny Moseley and Band Hero are up to date in the online PCL. I'll fix Formula One 2001 later tonight

Thanks, but the real place to put that info (or just "sync with the PCL") is at the commit message ;)

**Comment by refractionpcsx2** [comment]
we can always add that to the commit merge message :)

**Comment by avih** [comment]
I prefer to not mess with the original author's message, both out of courtesy and out of laziness. I'm glad they happen to align here ;)

**Comment by avih** [comment]
Note that I just updated GameIndex.dbf from the online compatibility list here 4096e72835e434dc6a72606a5a32291b6d989253 .

If you have further updates, please send PR only after the online compatibility list already has that info. Thanks.

**Comment by avih** [comment]
@MrCK1 status?

**Comment by MrCK1** [comment]
Sorry I'll try to iron this out tomorrow when I'm off work ;)

**Comment by MrCK1** [comment]
@avih I've let this PR sit for too long 😛

Is this ready to go?

**Comment by refractionpcsx2** [comment]
That's fine :) I'll merge now, if there's anything else I'm sure avih can reply :)

**Comment by avih** [comment]
> Anything else you need me to do?

Ops, seems like I missed your message at the time. Sorry :)

Anyway, to the point, unfortunately you didn't do what I requested, specifically:

You updated those but they're still not at our online compatibility list:
- SCPS-15019 / Name   = Formula One 2001
- SLES-50362 /  Name   = Jonny Moseley Mad Trix
- SLES-50619 /  Name   = Jonny Moseley Mad Trix
- SLES-55578 /  Name   = Band Hero
- SLPS-20120 /  Name   = Formula One 2001

In fact, of all the games which you updated only these are at our online list:
- SCUS-97150 / Name   = Formula One 2001
- SLUS-20229 / Name   = Jonny Moseley - Mad Trix
- SLUS-21898 / Name   = Band Hero

And your commit message also doesn't explain why the patch was removed - which I requested that you do.

**Comment by avih** [comment]
@MrCK1
- Please make sure the missing entries make it to our online compatibility list.
- In the future, don't post other Games DB PRs before the online list already have those updated.

**Comment by avih** [comment]
@bositman could you please help @MrCK1 with the missing online compatibility info? Also, don't take them automatically, I.e. we need the compatibility forum threads with data of who tested it, on what systems, etc.

**Comment by bositman** [comment]
If they aren't in the online compatibility list, we don't have a forum thread for them here: http://forums.pcsx2.net/Forum-Public-compatibility-list
So someone has to follow this for each game: http://forums.pcsx2.net/Thread-Request-thread-for-new-game-in-compatibility-list

**Comment by avih** [comment]
Well.. if we don't have threads for them then I assume @MrCK1 tested all of them personally, and so he should create these threads.

If he won't, I'll revert this commit, because we want to have compatibility info at the DB only where we have some records that this game actually works at the level the DB claims.

We already have 263 games at the `GameIndex.dbf` which are not at our online compatibility list. 5 of those are from this commit.

**Comment by MrCK1** [comment]
I guess I forgot about the rest of the world :p

**Comment by avih** [comment]
@MrCK1 can we expect you to handle it soon?, say, finish it within two weeks or so?

**Comment by MrCK1** [comment]
So for Band Hero, JMMT, and F1 2001 they need compatibility threads for non NTSC regions? Do I need to add the missing regions you mentioned above to the `GameIndex.dbf` or is that automatic?

I'm not great with Github so I'm not sure if I'll be able to get the commit message updated w/ the reason for the patch removal.

**Comment by avih** [comment]
This is not about github at all. This is about us having some kind of record that these games actually have the compatibility you claim they have.

How do you know all those games have a 5 compatibility level? We don't have any records for it on our compatibility forum, so I assume you tried all of them yourself and concluded that they all play perfectly - all versions and regions which you updated?

If you have tested all these regions and versions yourself, then please add these record following bositman's links above. And if you didn't test them yourself, why did you update their compatibility levels?

**Comment by MrCK1** [comment]
Sorry for the confusion. I didn't realize what I was doing until I took a look at the file itself. 

I made some stupid assumption that other regions needed to match (some compatibility info was also pre-existing I think) which was why I changed them. I shouldn't have left it that way; and it won't happen in the future.

@avih To answer your question, no I didn't test the other regions. Only the 3 that I updated in the compatibility forum were tested by myself.

**Comment by avih** [comment]
No worries, sh*t happens. In that case, I'll revert the updates which were not tested, and once/if the compatibility forum gets these serials, we'll update it again.

> skip video patches aren't needed anymore and game was fixed in ad61503

@prafullpcsx2 are you positive about that (considering @refractionpcsx2 being doubtful), or in general that the entire patch is not required anymore?

If not, shall we restore these patches too?

**Comment by FlatOutPS2** [comment]
The patch hasn't worked for years, so you can safely remove it. And the fix does work. ;)

**Comment by refractionpcsx2** [comment]
I was only doubtful about my ability to fix things :P Although to be fair it probably wasn't me who fixed it lol.

**Comment by FlatOutPS2** [comment]
You changed the flag from stop to pause, and that fixed the issue. :p

**Comment by refractionpcsx2** [comment]
oh that! cool :)

**Comment by avih** [comment]
OK, so I'll keep it without the patch and revert the updates to serials which are not at the online list. Thanks.

**Comment by avih** [comment]
> In fact, of all the games which you updated only these are at our online list:
> 
>     SCUS-97150 / Name = Formula One 2001
>     SLUS-20229 / Name = Jonny Moseley - Mad Trix
>     SLUS-21898 / Name = Band Hero

Turned out that except the Formula1 update, the rest were already updated anyway earlier from the online compatibility list (and so the merge commit didn't touch them since they had the same value as at the patch).

Bottom line, after the reversal of the accidental updates, this PR only updated the compatibility of SCUS-97150 and removed its patch.

**Comment by MrCK1** [comment]
OK, thanks for the help ;)

**Pull request merged**